generator/src/main/java/com/linkedin/pegasus/generator/JavaCodeUtil.java

```diff
@@ -17,26 +17,22 @@
 package com.linkedin.pegasus.generator;
 
 
-import java.nio.file.Paths;
+import com.sun.codemodel.*;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 import javax.annotation.Generated;
 import java.io.File;
 import java.net.MalformedURLException;
 import java.net.URI;
 import java.net.URL;
 import java.net.URLClassLoader;
+import java.nio.file.Paths;
 import java.util.ArrayList;
 import java.util.Iterator;
 import java.util.List;
 import java.util.StringTokenizer;
 
-import com.sun.codemodel.JAnnotationUse;
```

Comment:
Revert these changes, pegasus codestyle doesn't allow wildcard imports (except for static methods)

Reply:
Oh okay. I should do that to all the files. These were done by IntelliJ.  I'll do it for all files. Do we have a linter for this that would catch this issue?

Reply:
reverted. 

I do have concern about needing to change the codestyle since IntelliJ natively did some of these changes without my intervention.